throw new Error('lel');

export const getNumber = () => Promise.resolve(3);
